src/components/ChangePassword.tsx

```diff
@@ -1,8 +1,89 @@
-import React from 'react'
-import ChangePassword from 'synapse-react-client/dist/containers/ChangePassword'
+import React, { useEffect, useState } from 'react'
+import { FormGroup, FormControl, FormLabel, Button } from 'react-bootstrap'
+import { ChangePasswordWithCurrentPassword } from 'synapse-react-client/dist/utils/synapseTypes/ChangePasswordRequests'
+import { displayToast } from 'synapse-react-client/dist/containers/ToastMessage'
+import { UserProfile } from 'synapse-react-client/dist/utils/synapseTypes'
+import { useSynapseContext } from 'synapse-react-client/dist/utils/SynapseContext'
+import { SynapseClient } from 'synapse-react-client'
 
-const ChangePasswordPage: React.FunctionComponent = () =>{
-    return(<ChangePassword/>)
+export type ChangePasswordProps = {
+    onCancel?: Function
 }
 
-export default ChangePasswordPage
\ No newline at end of file
+export const ChangePassword = (props: ChangePasswordProps) => {
+    const { accessToken } = useSynapseContext()
+    const [oldPassword, setOldPassword] = useState<string>('')
+    const [newPassword, setNewPassword] = useState<string>('')
+    const [confirmPassword, setConfirmPassword] = useState<string>('')
+    const [currentUserProfile, setUserProfile] = useState<UserProfile>()
+
+    useEffect(() => {
+        async function getUserProfile() {
+          const userProfile = await SynapseClient.getUserProfile(accessToken)
+          setUserProfile(userProfile)
+        }
+        if (accessToken) {
+          getUserProfile()
+        } else {
+          setUserProfile(undefined)
+        }
+      }, [
+        accessToken
+    ])
+
+    const handleChangePassword = async (clickEvent: React.FormEvent<HTMLElement>) => {
+        clickEvent.preventDefault()
+        try{
+            if(newPassword !== confirmPassword){
+                displayToast('New password and confirm password does not match.', 'danger')
```

Comment:
Just slight change to copy

```suggestion
                displayToast('Passwords do not match.', 'danger')
```

Bigger question, though, and I see this was just copy/pasted from SRC, but did the design call for this to be a toast message? Seems odd to display a toast here instead of using HTML5 form validation or Bootstrap [FormControl.Feedback](https://react-bootstrap-v4.netlify.app/components/forms/#form-control-feedback-props), so I would suggest considering that.

Reply:
There wasn't a specific design call for this. However, the registration page was also using a toast message which led me to using toast messages.